app/models/mixins/authentication_mixin.rb

```diff
@@ -13,6 +13,7 @@ module AuthenticationMixin
     virtual_delegate :authentication_status,
                      :to        => "authentication_status_severity_level.status",
                      :default   => "None",
+                     :type      => :string
```

Comment:
Missing a comma here ^

Reply:
huh - I pushed a fix for this. not sure why it is still showing up